Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,7 +84,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $ui = new SymfonyStyle($input, $output);
 
         /** @var ManagerRegistry $doctrine */
-        $doctrine = $this->getContainer()->get('doctrine');
+        $doctrine = $this->getDoctrine();
```

Comment:
I think your change makes line 86 useless, as well as this one time variable. This should probably be changed to `$em = $this->getDoctrine()->getManager($input->getOption('em'))`

Reply:
True. Done.